recipe/meta.yaml

```diff
@@ -1,16 +1,15 @@
-{% set version = "1.0.3" %}
+{% set version = "1.1.0" %}
 
 package:
   name: mpc
   version: {{ version }}
 
 source:
-  fn: mpc-{{ version }}.tar.gz                                              # [unix]
-  url: http://www.multiprecision.org/mpc/download/mpc-{{ version }}.tar.gz  # [unix]
-  sha256: 617decc6ea09889fb08ede330917a00b16809b8db88c29c31bfbb49cbf88ecc3  # [unix]
-  fn: mpc-{{ version }}.zip                                                 # [win]
-  url: https://github.com/isuruf/mpc/archive/v{{ version }}.zip             # [win]
-  sha256: acc13357573d2ba542d93d1506f905bc7781ba9396fb21c5772c66c7048d7c53  # [win]
+  fn: mpc-{{ version }}.tar.gz
+  url: http://www.multiprecision.org/mpc/download/mpc-{{ version }}.tar.gz
```

Comment:
This url is not valid anymore. Have to use `https://ftp.gnu.org/gnu/mpc/mpc-{{ version }}.tar.gz`